docs/websocket/pronunciation_analyses.md

```diff
@@ -22,6 +22,31 @@ nl.itslanguage.pronunciation.init_analysis
 The unique analysis id is returned as a string. The id is used in other calls
 to identify the current analysis.
 
+## Initialise pronunciation challenge
+
+Before performing the analysis, the reference audio of the challenge needs to
+be prepared. This involves aligning the audio. When the RPC is called, the
+challenge is initialised asynchronously. When the challenge is to be used, the
+server automatically waits for the challenge initialisation to finish. If the
+initialisation results in an error, the error is relayed to the client.
+
+### URI
+
+```
+nl.itslanguage.pronunciation.init_challenge
+```
+
+## Fetch reference alignment
+
+The challenge that will be done is registered with a reference audio file to
+compare your speech with. This will need to be fetched before being able to send
```

Comment:
Please do not refer to "your" speech, refer to something like "the student's speech".